README.md

```diff
@@ -44,17 +49,14 @@ async fn main() -> Result<(), Box<dyn std::error::Error + Send + Sync>> {
     // 'asiaphotos-2015.zip' to bucket 'asiatrip'.
     client
         .upload_object(
-            &mut UploadObjectArgs::new(
-                &bucket_name,
-                "asiaphotos-2015.zip",
-                "/home/user/Photos/asiaphotos.zip",
-            )
-            .unwrap(),
+            &mut UploadObjectArgs::new(&bucket_name, "asiaphotos-2015.zip", "/tmp/asiaphotos.zip")
```

Comment:
Can we use `"/home/user/Photos/asiaphotos.zip"` instead of `"/tmp/asiaphotos.zip"` like all other SDKs?

Reply:
Ok, I had changed it only to test it locally